versioned_docs/version-8.3/apis-tools/working-with-apis-tools.md

```diff
@@ -60,7 +60,7 @@ Camunda 8 components have APIs to enable polyglot developers to work with in the
 type:"link", href:"/docs/next/apis-tools/operate-api/overview/", label: "Operate API (REST)", docId:"apis-tools/operate-api/operate-api-overview"
 },
 {
-type:"link", href:"/optimize/next/apis-tools/optimize-api/optimize-api-authorization/", label: "Optimize API (REST)"
+type:"link", href:"/optimize/apis-tools/optimize-api/optimize-api-authorization/", label: "Optimize API (REST)"
```

Comment:
Should this be changed from `..../optimize-api-authorization` to `..../optimize-api-authentication`?

Reply:
No, it shouldn't! I made this comment before noticing the URL was only moved in `next`.